src/libAtomVM/ets.c

```diff
@@ -52,7 +58,7 @@ struct EtsTable
     term name;
     bool is_named;
     int32_t owner_process_id;
-    size_t keypos;
+    size_t key_index;
```

Comment:
This was already using 0-based indexes but was named `position` which suggests 1-based indexes. Cleaned all up.